src/coreclr/jit/gentree.cpp

```diff
@@ -18162,6 +18204,7 @@ FieldSeqNode::FieldSeqNode(CORINFO_FIELD_HANDLE fieldHnd, FieldSeqNode* next, Fi
     if (fieldHnd != NO_FIELD_HANDLE)
     {
         assert(JitTls::GetCompiler()->eeIsFieldStatic(fieldHnd) == IsStaticField());
+        // TODO-PhysicalVN: assert that "offset" is correct.
```

Comment:
Leaving this for a future PR?

Reply:
Yep, will add it as part of refactoring `FieldSeqNode` into more of a `FieldInfo`.